microsoft/AspNetCore-React-WebApp
# Upgrade DSX Project Template to .NET Core 6.0 and EF Core 6.0

Why?
Update backend Nuget packages in preparation for any new projects that will use the project template.

What changed?
This PR modifies the following files:

Project Template.API.csproj
---- nswag.json inside API.csproj
Project Template.Command.csproj
Project Template.Data.csproj
Project Template.Test.csproj

How to test?

1. Checkout branch 'upgrade_.NET_6.0'
2. Build Solution ---
3. Verify in Output - '========== Rebuild All: 4 succeeded, 0 failed, 0 skipped =========='
